articles/getting_to_know/howto/graphics/HowTo_Draw_Text.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Drawing Text with a Sprite
+description: Demonstrates how to import a SpriteFont into a project and to draw text using DrawString
+requireMSLicense: true

```

Comment:
What does this mean?  Is a Microsoft License required for fonts?  Is there a place that explains this?

Reply:
:D, this is the front matter for the doc, meaning the original source of the doc is from Microsoft, so we automatically put an MS license note at the end of the article :D